stdlib/array.gr

```diff
@@ -1159,6 +1264,8 @@ provide module Immutable {
   /**
    * An empty array.
    *
+   * @example Array.Immutable.empty()
```

Comment:
Not a function; also a better example may be `Array.Immutable.empty == Array.Immutable.fromList([])`